meta/internal/meta.pb.go

```diff
@@ -518,8 +525,8 @@ func (m *ShardOwner) GetNodeID() uint64 {
 }
 
 type ContinuousQueryInfo struct {
-	Name             *string `protobuf:"bytes,1,req" json:"Name,omitempty"`
-	Query            *string `protobuf:"bytes,2,req" json:"Query,omitempty"`
+	Name             *string `protobuf:"bytes,1,req,name=Name" json:"Name,omitempty"`
```

Comment:
I presume all these changes are backwards compatible with existing commands that will be in the Raft log.